src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/FilterOperator.java

```diff
@@ -37,12 +36,16 @@ private static void addComparisonOperator(FilterOperator filterOperator) {
       operatorMap.put(filterOperator.getOperator(), filterOperator);
     }
 
-    public static FilterOperator getComparisonOperator(String operator) {
-      final FilterOperator filterOperator = operatorMap.get(operator);
-      if (filterOperator == null)
-        throw new JsonApiException(
-            ErrorCode.UNSUPPORTED_FILTER_OPERATION, "Unsupported filter operator " + operator);
+    public static FilterOperator findComparisonOperator(String operator) {
+      return operatorMap.get(operator);
+    }
 
+    @Deprecated // since 1.0.2 use "findComparisonOperator" instead
```

Comment:
Bad form to make caller catch exception, better to use `find()`. May delete before merging if things work as expected.